api/bots/giphy/test_giphy.py

```diff
@@ -51,11 +51,10 @@ def test_bot(self):
         # This message calls `send_reply` function of BotHandlerApi
         keyword = "Hello"
         gif_url = "https://media4.giphy.com/media/3o6ZtpxSZbQRRnwCKQ/giphy.gif"
-        expectations = {
-            keyword: get_bot_response(gif_url)
-        }
-        self.check_expected_responses(
-            expectations=expectations,
+        self.assert_bot_response(
+            message = {'content': keyword},
+            response = {'content': get_bot_response(gif_url)},
```

Comment:
I'm not confident that the right API here involves a dict with 1 key in it, but this is fine for now.